### ISSUE TITLE
Version 0.14.x removed -latency-ms

### ISSUE THREAD

**Issue by xeroiv** [MajorMJR/factorio-server-manager#38]
when starting server it gives an error 

`unrecognised option --latency-ms`

**Comment by MajorMJR** [comment]
Fixed in    https://github.com/MajorMJR/factorio-server-manager/commit/f9694bb6bcfe15095104390f41e0c3dfb66b619d

I've made the latency option optional, if the user selects a different value in the UI then the server will be started with the --latency-ms option.

I tested this with the latest 0.14.4 server version.  From the changelog they removed this option since 0.14, multiplayer performance seems to be improving nicely.

**Comment by xeroiv** [comment]
Ok that is probably a better solution than I had hacked together.  However the --latency-ms option appears to be removed from the arguments altogether.

`General options:
  -h [ --help ]                 display help
  --version                     show version information
  -v [ --verbose ]              enable verbose logging
  -c [ --config ] PATH          config file to use
  --no-log-rotation             don't rotate log file
  --mod-directory PATH          Mod directory to use

Running options:
  -s [ --map2scenario ] arg     map to scenario conversion
  -m [ --scenario2map ] arg     scenario to map conversion
  --apply-update arg            immediately apply update package
  --create FILE                 create a new map
  --map-gen-settings FILE       Map generation settings for use with --create. See data/map-gen-settings.example.json
  --start-server FILE           start a multiplayer server
  --start-server-load-latest    start a multiplayer server and load the latest available save
  --until-tick TICK             run a save until given map tick
  --mp-connect ADDRESS          start factorio and connect to address
  --load-game FILE              start Factorio and load a game in singleplayer
  --benchmark FILE              load save and run benchmark
  --benchmark-ticks N (=1,000)  number of ticks for benchmarking. Default is 1000
  --force-opengl                use OpenGL for rendering
  --force-d3d                   use Direct3D for rendering

Server options:
  --autosave-interval N (=2)    server autosave interval, in minutes
  --autosave-slots N (=3)       server autosave slots
  --allow-commands arg (=true)  allow (true), disallow (false), or restrict (admins-only) use of the command console
  --no-auto-pause               don't automatically pause when no players are
                                connected
  --port N                      network port to use
  --rcon-port N                 Port to use for RCON
  --rcon-password PASSWORD      Password for RCON
  --server-settings FILE        Path to file with server settings. See data/server-settings.example.json`

On an unrelated note, is it possible to add the server default port, autosave interval, and autosave number to the config file rather than hardcoded into ServerCtl.jsx?

**Comment by Curtisdhi** [comment]
I'm not sure if i'm doing soemthing wrong... but I get this error while trying to execute the binary
`
pickle@s:/factorioServers/factorio-server-manager$ ./factorio-server-manager --dir /factorioServers/factorio --port 8888
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x5fe43e]

goroutine 1 [running]:
panic(0x938080, 0xc8200100c0)
        /usr/lib/go-1.6/src/runtime/panic.go:481 +0x3e6
github.com/syndtr/goleveldb/leveldb.(*DB).setClosed(0x0, 0xc91d00)
        /home/chicks/go/src/github.com/syndtr/goleveldb/leveldb/db_state.go:220 +0x4e
github.com/syndtr/goleveldb/leveldb.(*DB).Close(0x0, 0x0, 0x0)
        /home/chicks/go/src/github.com/syndtr/goleveldb/leveldb/db.go:1027 +0x58
github.com/apexskier/httpauth.NewLeveldbAuthBackend(0xc820011a60, 0xc, 0x0, 0x0, 0x0, 0x7f8405e75028, 0xc820011ac0)
        /home/chicks/go/src/github.com/apexskier/httpauth/leveldbAuthBackend.go:36 +0x28d
main.(*AuthHTTP).createAuthDb(0xc82000abe0, 0xc820011a60, 0xc, 0x0, 0x0)
        /home/chicks/factorio-server-manager/src/auth.go:30 +0x81
main.main()
        /home/chicks/factorio-server-manager/src/main.go:85 +0x147
`
Perhaps I am just compiling this wrong?

**Comment by MajorMJR** [comment]
Did you build the binary or is it the compiled one in the release?  It looks like there is some panic getting generated when it is trying to create the leveldb files.  Maybe check if there are write permissions in the directory you are starting the server manager from.

**Comment by Curtisdhi** [comment]
I compiled it from the source as there isn't a compiled version for factorio 0.14.
I chmod everything with 777 perms to make sure it wasn't a permissions issues.
Though, I did compiled this on a different machine. and the error is refering to "/home/chicks" which doesn't exist on my server i am trying to execute, but that shouldn't be a problem?

**Comment by MajorMJR** [comment]
Check your conf.json file, maybe there is path defined there for database_file?  Just guessing since the server is looking in the /home/chicks directory on startup.

**Comment by Curtisdhi** [comment]
```
{
    "username": "admin",
    "password": "factorio",
    "database_file": "auth.leveldb"
}
```
Looks like the default config.json

Oh.. .umm just took a wild guess, and deleted the auth.leveldb directory, and it works fine now... 
strange. Well, thanks anyway.

**Comment by MajorMJR** [comment]
Awesome I guess that's a workaround unless we can find what caused database files to crash the program.

**Comment by Curtisdhi** [comment]
Yeah, i didn't think to copy the files before deleting them...

**Comment by NQRott** [comment]
any chance to upload the latest binary for 0.14. 
i tried to build it for 2 days in my ubuntu, it is always missing files from github.

**Comment by Curtisdhi** [comment]
@NQRott Yeah, i had the same problem. I getting those.
Use `go get "url"` replace url with the github url it says is missing.

**Comment by MajorMJR** [comment]
Yeah you will have to 'go get' all the third party libraries, check the .travis.yml file for a list of the libraries URL's.

I'll build a release tonight when I get home this evening after work.

**Comment by NQRott** [comment]
thx now i can start the server. it was compiled with the wrong name. i just replaced it with the old binary.

**Comment by MajorMJR** [comment]
I've uploaded the latest release with the bug fixes for 0.14.

https://github.com/MajorMJR/factorio-server-manager/releases/latest
